tests/s3/versioning/test_delete_object_tagging.py

```diff
@@ -348,3 +348,177 @@ def test_del_obj_tag_40436(self):
+        get_tag = resp[1]
+        latest = self.versions[self.object_name]["version_history"][-1]  # Get the latest Version ID
+        put_tag = self.ver_tag[self.object_name][latest]
+        assert_utils.assert_equal(get_tag, put_tag, "Mismatch in tag Key-Value pair."
+                                                    f"Expected: {put_tag} \n Actual: {get_tag}")
+
+        LOGGER.info("Step 8: GET Object Tagging for %s (without versionId specified)", self.object_name)
+        resp = s3_ver_tlib.get_object_tagging(s3_tag_test_obj=self.s3_tag_obj,
+                                              s3_ver_test_obj=self.s3_ver_obj,
+                                              bucket_name=self.bucket_name,
+                                              object_name=self.object_name)
+        assert_utils.assert_true(resp[0], resp)
+        # expecting "TagSet": []
+        assert_utils.assert_false(resp[1], resp)
+
+        LOGGER.info("Step 9: DELETE Object Tagging for %s with versionId=%s", self.object_name,
+                    ver_list[0])
+        resp = s3_ver_tlib.delete_object_tagging(bucket_name=self.bucket_name,
+                                                 s3_ver_test_obj=self.s3_ver_obj,
+                                                 s3_tag_test_obj=self.s3_tag_obj,
+                                                 object_name=self.object_name,
+                                                 version_id=ver_list[0])
+        assert_utils.assert_true(resp[0], resp)
+
+        LOGGER.info("Step 10: GET Object Tagging for %s with versionId=%s", self.object_name,
+                    ver_list[0])
+        resp = s3_ver_tlib.get_object_tagging(s3_tag_test_obj=self.s3_tag_obj,
+                                              s3_ver_test_obj=self.s3_ver_obj,
+                                              bucket_name=self.bucket_name,
+                                              object_name=self.object_name,
+                                              version_id=ver_list[0])
+        assert_utils.assert_true(resp[0], resp)
+        get_tag = resp[1]
+        put_tag = self.ver_tag[self.object_name][f"{ver_list[0]}"]
+        assert_utils.assert_equal(get_tag, put_tag, "Mismatch in tag Key-Value pair."
+                                                    f"Expected: {put_tag} \n Actual: {get_tag}")
+
+        LOGGER.info("Step 11: GET Object Tagging for %s (with versionId=null)", self.object_name)
+        resp = s3_ver_tlib.get_object_tagging(s3_tag_test_obj=self.s3_tag_obj,
+                                              s3_ver_test_obj=self.s3_ver_obj,
+                                              bucket_name=self.bucket_name,
+                                              object_name=self.object_name,
+                                              version_id=ver_list[1])
+        assert_utils.assert_true(resp[0], resp)
+        # expecting "TagSet": []
+        assert_utils.assert_false(resp[1], resp)
```

Comment:
We need to verify null version id's tagset is intact

Reply:
Its expecting empty tag list as per test steps , verified as 
# expecting "TagSet": []
assert_utils.assert_false(resp[1], resp)